kernelci/rootfs.py

```diff
@@ -44,14 +43,29 @@ def _build_debos(name, config, data_path):
             test_overlay=config.test_overlay,
             crush_image_options=" ".join(config.crush_image_options)
             )
-        shell_cmd(cmd)
+    shell_cmd(cmd)
 
     return True
 
 
-def build(name, config, data_path):
+def build(name, config, data_path, arch=None):
```

Comment:
I think it's better to just require `arch` as an argument, so each call to `build()` only produces one rootfs.  It simplifies the code and makes the behaviour more deterministic.